.vscode/launch.json

```diff
@@ -24,8 +25,10 @@
                 "tdd",
                 "--timeout",
                 "999999",
+                "--compilers",
+                "js:@babel/register",
                 "--colors",
-                "${file}"
+                "${file}",
```

Comment:
Hmm - this extra `,` makes this invalid JSON!

Reply:
I think VS Code handles trailing commas. This was probably prettier before I got it ignoring the .vscode folder. I'll fix it up.